src/pages/academia/educators.js

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import Layout from '../../components/dev-hub/layout';
-import { Helmet } from 'react-helmet';
 import { useSiteMetadata } from '../../hooks/use-site-metadata';
 import {
     TopBanner,
     HowToJoin,
     ProgramBenefits,
 } from '../../components/pages/educators';
+import PageHelmet from '~components/dev-hub/page-helmet';
+
+const PAGE_DESCRIPTION =
+    'MongoDB for Academia is for educators who want to prepare students for careers that require in-demand database skills that power modern applications.';
 
 export default () => {
-    const metadata = useSiteMetadata();
+    const { title, siteUrl } = useSiteMetadata();
     return (
         <Layout>
-            <Helmet>
-                <title>Academia for Educators - {metadata.title}</title>
-            </Helmet>
-            <TopBanner />
+            <PageHelmet
+                canonicalUrl={`${siteUrl}/academia/educators/`}
```

Comment:
I see most of these use `siteUrl` for the canonical url which is correct. Would a better set of props for `PageHelmet` include just using the page path and then adding the `siteUrl` within `PageHelmet` to avoid the need to use it in every component which has `PageHelmet` as a child?

Reply:
A similar thought is that many `title`s use the site title as well. Should we also append this automatically in `PageHelmet`?

Reply:
Okay sounds good to me if we leave `title` more generic here and continue to use from the calling component, that works for me.

Similar thought with the `siteUrl`. I don't think we would need to replace this for any of the main pages.